src/test/java/com/wb/test/WeatherForecastApiApplicationTests.java

```diff
@@ -0,0 +1,78 @@
+package com.wb.test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+
+import com.wb.controller.WeatherForecastController;
+import com.wb.service.WeatherService;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import com.wb.payload.Weather;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class WeatherForecastApiApplicationTests {
```

Comment:
niestety testy do przepisania (a wlasciwie napisania bo tutaj mamy 2 i to bledne). ale najpierw troche poczytaj o testach jednostkowych w javie. cos swiezego zebys mial junit 5. nie ma co na slepo pisac bo nic z tego nie bedzie